controllers/extension-shoot-dns-service/charts/extension-shoot-dns-service/values.yaml

```diff
@@ -24,3 +24,10 @@ resources:
 controllers:
   concurrentSyncs: 5
   ignoreOperationAnnotation: false
+
+imageVectorOverwrite: |
```

Comment:
Please, comment the sample value, for example see how it is done for `provider-gcp` https://github.com/gardener/gardener-extensions/blob/639d73ebda96b7c1e54f4df07c2831ed4c1cb52d/controllers/provider-gcp/charts/provider-gcp/values.yaml#L25-L36



Otherwise this it will always overwrite the image defined here https://github.com/gardener/gardener-extensions/blob/639d73ebda96b7c1e54f4df07c2831ed4c1cb52d/controllers/extension-shoot-dns-service/charts/images.yaml

Reply:
good catch. Forget to comment this part after testing. 😄